simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -183,7 +192,7 @@
 
       <sensor name="velodyne_lidar" type="ray">
         <visualize>false</visualize>
-        <pose>-0.277622 0 0.648208 0.0 0.0 0.0</pose>
+        <pose> -0.277622 0.0 0.648208 0.0 -0.0 0.0 </pose>
```

Comment:
Could you adjust the down camera such that it is 0.1m behind the new camera position? Also, I was not entirely sure about rotation, so please double check me on it.